codespell_lib/data/dictionary.txt

```diff
@@ -23340,13 +23415,31 @@ exproted->exported
 exproting->exporting
 exprots->exports
 exprted->exported
+exprting->exporting
+exprts->exports, experts, expats,
+exptect->expect
+exptectation->expectation
+exptectations->expectations
 exptected->expected
+exptecting->expecting
+exptects->expects
 exra->extra
-exract->extract
+exract->extract, exact,
+exracted->extracted, exacted,
+exracting->extracting, exacting,
+exraction->extraction
+exractions->extractions
+exractor->extractor
+exractors->extractors
+exracts->extracts, exacts,
 exrension->extension
 exrensions->extensions
+exress->express, excess,
 exressed->expressed
+exresses->expresses
```

Comment:
Again `egresses` and more importantly `excesses`?

Reply:
Added `excesses`